packages/sync/src/useMutltiplayerDemo.ts

```diff
@@ -43,7 +51,18 @@ function getEnv(cb: () => string | undefined): string | undefined {
 const DEMO_WORKER = getEnv(() => process.env.TLDRAW_BEMO_URL) ?? 'https://demo.tldraw.xyz'
 const IMAGE_WORKER = getEnv(() => process.env.TLDRAW_IMAGE_URL) ?? 'https://images.tldraw.xyz'
 
-/** @public */
+/**
+ * Creates a tldraw store synced with a multiplayer room hosted on tldraw's demo server `https://demo.tldraw.xyz`.
+ *
+ * The store can be passed directly into the `<Tldraw />` component to enable multiplayer features.
+ * It will handle loading states, and enable multiplayer UX like user cursors and following.
+ *
+ * Any data you save will be publicly accessible to anyone who knows the room ID, and any stored data is wiped after a day or so.
+ *
```

Comment:
```suggestion
 *
 * @example
 * ```tsx
 * function MyApp() {
 *     const store = useMultiplayerDemo({roomId: 'my-app-test-room'})
 *     return <Tldraw store={store} />
 * }
 * ```
 *